package.json

```diff
@@ -196,14 +196,12 @@
     "uglify-js": "^3.4.0",
     "url-loader": "^0.6.2",
     "webpack": "^4.1.1",
-    "webpack-bundle-analyzer": "^2.11.1",
+    "webpack-bundle-analyzer": "^3.3.2",
     "webpack-dev-server": "^3.1.1",
     "webpack-manifest-plugin": "^2.0.0-rc.2",
     "winston": "^2.2.0"
   },
-  "optionalDependencies": {
-    "fsevents": "^1.0.7"
```

Comment:
This is one of the oldest parts of our package.json: https://github.com/department-of-veterans-affairs/vets-website/commit/6bf2554ed442db7fb3db06c2c750b5fe6282e772

It does not appear necessary anymore, since we're not using shrinkwrap.